src/layouts/business.vue

```diff
@@ -13,7 +13,7 @@ const crumbConstructors = computed(() => {
 const systemMessage = ref('')
 onMounted(async () => {
   await useBcrosLaunchdarkly().ldClient.waitUntilReady()
-  systemMessage.value = useBcrosLaunchdarkly().getStoredFlag('banner-text')
+  systemMessage.value = (useBcrosLaunchdarkly().getStoredFlag('banner-text') || '').trim()
```

Comment:
fixes banner showing when text is blank (entities manages their ld flags a bit differently so they have empty text instead of undefined)